pkg/util/mount/mount_windows.go

```diff
@@ -0,0 +1,205 @@
+// +build windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mount
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"k8s.io/utils/exec"
+
+	"github.com/golang/glog"
+)
+
+type Mounter struct {
+	mounterPath string
+}
+
+// New returns a mount.Interface for the current system.
+// It provides options to override the default mounter behavior.
+// mounterPath allows using an alternative to `/bin/mount` for mounting.
+func New(mounterPath string) Interface {
+	return &Mounter{
+		mounterPath: mounterPath,
+	}
+}
+
+func (mounter *Mounter) Mount(source string, target string, fstype string, options []string) error {
+	target = normalizeWindowsPath(target)
+
+	if source == "tmpfs" {
+		glog.Infof("azureDisk: mounting source (%q), target (%q), with options (%q)", source, target, options)
+		return os.MkdirAll(target, 0755)
+	}
+
+	parentDir := filepath.Dir(target)
+	if err := os.MkdirAll(parentDir, 0755); err != nil {
+		return err
+	}
+
+	glog.V(4).Infof("azureDisk: mount options(%q) source:%q, target:%q, fstype:%q, begin to mount",
+		options, source, target, fstype)
+	bindSource := ""
+	ex := exec.New()
+
+	if bind, _ := isBind(options); bind {
+		bindSource = normalizeWindowsPath(source)
+	} else {
+		// mount azure file
+		if len(options) < 2 {
+			glog.Warningf("azureDisk: mount options(%q) command number(%d) less than 2, source:%q, target:%q, skip mounting",
```

Comment:
here too

Reply:
this code only runs on windows, so azureDisk has more meaning than windowsDisk.

Reply:
correct, I have changed to azureMount, is that ok?